Gemfile.lock

```diff
@@ -315,7 +315,7 @@ GEM
       activesupport (>= 3.0)
       i18n
       polyamorous (~> 1.3.2)
-    redis (4.0.1)
+    redis (3.3.5)
```

Comment:
Why not locking the version in the `Gemfile` to make it explicit, maybe with a comment?

Reply:
@MarcoBgn Because it's a dependency of a dependency, that's not what you should use the `Gemfile` for. And in my experience when you do that, it's going to stay in the `Gemfile` for no reason for ever.

Reply:
@your I cannot find any substantial info about:

> Because it's a dependency of a dependency, that's not what you should use the Gemfile

It seems way clearer to me, similarly to:
https://stackoverflow.com/questions/7434263/can-i-force-a-gems-dependencies-in-gemfile

Anyway, the commit message is very clear, so either way will be easily trackable

Reply:
Discussed offline, summarising:

The linked answer is quite generic, but that's exactly the approach followed to bump the version here, **plus** a final step of removing the explicit dependency from the `Gemfile` in the end, which would still allow you to let bundle find the best version to use for `redis` (which again, is not a direct dependency) if you run it again.

We agreed that adding it as a direct dependency for the time being is probably more *"developer (or human)-friendly"* though.